enhancements/network/cloud-egress-ip.md

```diff
@@ -0,0 +1,445 @@
+---
+title: TBD - for now: Cloud API component for egress IP
+authors:
+  - "@alexanderConstantinescu"
+reviewers:
+  - "@danwinship"
+  - "@abhat"
+  - "@squeed"
+  - "@tssurya"
+  - "@trozet"
+approvers:
+  - TBD
+creation-date: 2020-12-16
+last-updated: -
+status: implementable
+---
+
+# TBD - for now: Cloud API component for egress IP
```

Comment:
Why not simply call it "Cloud Config Component"?

Reply:
As I mentioned in the doc: we're not sure what this component will be at this point, so I left it open ended for that reason. I could change the name, though I am not convinced it better indicates what the component does / would do.

Reply:
needs something "networking-ish" in the name. There are other cloud-related operators